instrumented/nightly-tests/src/main/kotlin/com/datadog/android/nightly/fragments/ViewTrackingFragment.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.nightly.fragments
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.datadog.android.nightly.R
+import com.datadog.android.nightly.utils.measure
+
+internal class ViewTrackingFragment : Fragment() {
+
+    val testMethodName = "rum_fragment_view_tracking_strategy"
```

Comment:
unused and also `measure` below seems redundant. Same applies to other fragment classes as well.

Reply:
It is not redundant as we are actually measuring if what we do under the hood in the `AndroidXFragmentLifecycleCallbacks` is not taking more than `24ms`